### ISSUE TITLE
Issues when running in deno deploy

### ISSUE THREAD

**Issue by eliassjogreen** [lucacasonato/deno_aws_sign_v4#13]
I am getting this error when running deploying to deploy (not present when running locally in deployctl) probably because of the fetch rewrite of 1.9.1:
```
❗️    TypeError: This property is not implemented.
    at Request.get cache (deno:op_crates/fetch/23_request.js:343:13)
    at AWSSignerV4.sign (https://deno.land/x/aws_sign_v4@1.0.0/mod.ts:86:28)
    at S3Bucket._doRequest (https://deno.land/x/s3@0.4.0/src/bucket.ts:26:50)
    at S3Bucket.getObject (https://deno.land/x/s3@0.4.0/src/bucket.ts:152:32)
    at unknownHandler (https://raw.githubusercontent.com/denosaurs/crux.land/9069446ebf0b9fe1ab17f61e72d595a4e31a64f0/main.ts:40:35)
    at https://raw.githubusercontent.com/denosaurs/crux.land/9069446ebf0b9fe1ab17f61e72d595a4e31a64f0/util/router.ts:7:24
    at https://raw.githubusercontent.com/denosaurs/crux.land/9069446ebf0b9fe1ab17f61e72d595a4e31a64f0/main.ts:82:7
    at innerInvokeEventListeners (deno:op_crates/web/02_event.js:750:27)
    at invokeEventListeners (deno:op_crates/web/02_event.js:785:5)
    at dispatch (deno:op_crates/web/02_event.js:660:11)
```
This could technically also be an issue with s3.

**Comment by lucacasonato** [comment]
> not present when running locally in deployctl

Even in 1.9.1/1.9.2?

Anyway, fix coming up in #14

**Comment by lucacasonato** [comment]
Fixed in #14
